src/Socket/messages-recv.ts

```diff
@@ -8,6 +8,11 @@ import { makeChatsSocket } from "./chats"
 import { extractGroupMetadata } from "./groups"
 
 const getStatusFromReceiptType = (type: string | undefined) => {
+
+    if(type === 'played') {
+        return proto.WebMessageInfo.WebMessageInfoStatus.PLAYED;
+    }
+
     if(type === 'read' || type === 'read-self') {
         return proto.WebMessageInfo.WebMessageInfoStatus.READ
     }
```

Comment:
As @topkek on discord explained, this is better:
```suggestion
    if(type === 'read' || type === 'read-self') {
        return proto.WebMessageInfo.WebMessageInfoStatus.READ
    } else if(type === 'played') {
        return proto.WebMessageInfo.WebMessageInfoStatus.PLAYED;
    }
```

(there is always only 1 type of ack :shrug: )